### PR TITLE
feat : added modal styling

### PR DESCRIPTION
**Pull request by raman1999**
image

### CODE REVIEW

File: components/modal.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgba(0, 0, 0, 0.4);
+    
+  }
+  
+  /* Modal box */
+  .modal-content {
+    margin: auto;
```
Comment (ttahm3d) [inline components/modal.css:18]: give margin: 0 auto; if you want to center it

**Review by ttahm3d — APPROVED** [review]
looks good. Have suggeste a minor change

File: components/modal.css (hunk 1 above)
Comment (raman1999) [reply components/modal.css:18]: Thanks for suggestion , but I prefer its position there only, and actually content is inside modal container so by changing padding-top  of container we can center it.

**Review by therajatg — APPROVED** [review]
Looks clean, you can go ahead and merge.

**Pull request merged**